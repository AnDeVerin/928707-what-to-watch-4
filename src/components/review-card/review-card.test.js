import React from 'react';
import renderer from 'react-test-renderer';
import ReviewCard from './review-card.jsx';

const review = {
  text: `Bla-bla-bla. This is some review test`,
  author: `Doctor Who`,
  date: `2020-06-28`,
  rating: 7.7,
};

describe('ReviewCard component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<ReviewCard review={review} />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
