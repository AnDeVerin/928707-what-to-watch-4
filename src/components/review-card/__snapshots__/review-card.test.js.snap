// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewCard component renders correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Bla-bla-bla. This is some review test
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Doctor Who
      </cite>
      <time
        className="review__date"
        dateTime="2020-06-28"
      >
        June 28, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    7.7
  </div>
</div>
`;
